feconf.py

```diff
@@ -403,6 +403,13 @@ def get_empty_ratings():
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
 ENABLE_STRING_CLASSIFIER = False
+SHOW_COLLECTION_NAVIGATION_TAB_MAIN = False
```

Comment:
This probably isn't needed. The main tab isn't experimental; it should always be available (and the editor gets a bit weird when you visit it with all of these turned off).